tox.ini

```diff
@@ -1,38 +1,42 @@
 [tox]
 envlist =
-    py{36,37,38}-django{22,30,31},
-    py{36,37,38,39}-djangodev
+    py{36,37,38}-dj{22,30,31},
+    py{36,37,38,39}-djdev
```

Comment:
Should this be just `djmaster`, similar as in other projects?

Reply:
Hm, in theory yes, but given that `master` isn't really a version specifier, I think `dev` is actually more accurate. I'm gonna consider this to be a lax issue.